igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/service/GitlabCiPipelineUtils.java

```diff
@@ -25,7 +25,7 @@ public static GenericBuild genericBuild(
     String friendlyName = String.format("%s (%s)", pipeline.getRef(), pipeline.getId());
     GenericBuild genericBuild = new GenericBuild();
     genericBuild.setBuilding(GitlabCiResultConverter.running(pipeline.getStatus()));
-    genericBuild.setNumber(pipeline.getId());
+    genericBuild.setNumber(pipeline.getId() + "");
```

Comment:
Maybe `String.valueOf(pipeline.getId())` is the idea is to transform an int to a string?

Reply:
agreed, change made.